csrankings.csv

```diff
@@ -3638,6 +3642,7 @@ Eisa Zarepour,UNSW,https://research.unsw.edu.au/people/dr-eisa-zarepour,MgSbGaYA
 Eitan Grinspun,Columbia University,http://www.cs.columbia.edu/~eitan,0_vvBZkAAAAJ
 Eitan Yaakobi,Technion,http://www.cs.technion.ac.il/people/yaakobi,Viv2E9AAAAAJ
 Ekaterina Lebedeva,Australian National University,https://cecs.anu.edu.au/people/ekaterina-lebedeva,3Ey9ZWIAAAAJ
+Ekaterina Shutova,University of Amsterdam,https://www.cl.cam.ac.uk/~es407/,jqOFBGoAAAAJ

```

Comment:
Is there a web page for this person at Amsterdam?

Reply:
yes at the institute (albeit her
personal page is still hosted at cam)

I've updated it